libs/shared/flagd-core/src/lib/feature-flag.ts

```diff
@@ -1,12 +1,12 @@
-import {FlagValue} from '@openfeature/server-sdk';
+import {FlagValue} from '@openfeature/core';
 
 /**
  * Flagd flag configuration structure mapping to schema definition.
  */
 export interface Flag {
   state: string,
   defaultVariant: string,
-  variants: {[key: string]: FlagValue},
+  variants: { [key: string]: FlagValue },
   targeting: string
```

Comment:
```suggestion
  targeting?: string;
```

Targeting should be optional.